### DISCUSSION TITLE
Newbie trying to place an image within node.

### DISCUSSION THREAD

**Discussion by MickyLuv** [freeplane/freeplane#1538] [Help]
I'm using the Buzan template because it's the style I'm most familiar with from hand drawn maps.
The central node has an icon in it that appears to represent an image placement. 
When I try to inset an image in the node to overwrite this "placeholder?" the image gets placed outside the node.
I watched a video on placing an image inside a node and tried the method on a child node. The image once again was placed outside the node. The image is a 119 x 119  Pixels (1:1) bmp file.
Please can you tell me where I can find instructions to do this effectively?

**Comment by i-plasm** [comment]
Hi, the icon you see is not a place holder, but the icon generated by the hyperlink set to the node (menuitem:_ExternalImageAddAction), which you can remove if you'd like at ` Insert -> Link -> Remove link `. What you experienced is the normal behavior of the Add Image action. 
There exists an option for inserting an image inside the core (but it seems if the core already has contents they will be deleted): ` Edit -> Node core -> Image by choice or link ` . You can edit the core afterwards in  ` Edit -> Edit node core in dialog `. Another way of inserting an image, but which allows you to paste it without deleting the node core, is:
1. Right click on the node, and choose ` Edit -> Edit node core in dialog `
1. Open your image in a browser (e.g Chrome, Firefox).
1. Right click on the image displayed on the browser, and choose "copy image"
1. Now you can paste the image on the dialog

**Reply by MickyLuv** [reply]
Thanks very much for the help. I was out of ideas. Now I can plod on.
